index.html

```diff
@@ -195,18 +187,26 @@ <h6 class="mb-0">&copy 2020 GeyserMC</h6>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
         <script>
-            $(document).ready(function(){
-              $(window).scroll(function() {
+            var gmc_nav_listener = function() {
                 if ($(document).scrollTop() > 225) {
                   $(".navbar").css("background-color", "rgba(0, 0, 0, 0.9)");
+                  $(".navbar").css("box-shadow", "0 .5rem 1rem rgba(0, 0, 0, 0.15)");
                   $(".navbar").css("margin", "0px");
-                  $(".navbar").css("padding", "20px 20px 10px 20px");
+                  $(".navbar").css("padding", "10px 10px 10px 10px");
+                  $(".navbar").css("border-radius", "0");
+                  $('.nav-logoimg').css("opacity", '1.0');
                 } else {
-                  $(".navbar").css("background-color", "rgba(255, 255, 255, 0.1)");
+                  $(".navbar").css("background-color", "rgba(255, 255, 255, 0.0)");
+                  $(".navbar").css("box-shadow", "0 .5rem 1rem rgba(0, 0, 0, 0.0)");
                   $(".navbar").css("margin", "10px");
-                  $(".navbar").css("padding", "10px 10px 10px 10px");
+                  $(".navbar").css("padding", "5px 10px 5px 10px");
+                  $(".navbar").css("border-radius", "25px");
+                  $('.nav-logoimg').css("opacity", '0.0');
```

Comment:
Can `.navbar` and `.nav-logoimg` be pulled into variables to prevent searching for the element every time?